### ISSUE TITLE
consider changing TestSubscriber.sequence to something else

### ISSUE THREAD

**Issue by heckj** [tcldr/Entwine#5]
In using TestSubscriber, I found the property name `sequence` to be confusing. While it makes sense in the framing of what's been collection, and that it's an ordered list, I immediately confused it for Combine's Sequence.

I wanted to suggest that perhaps a different name would be easier to work with - although I'm lacking on brilliant ideas. My fogged brain only really came up with `results` or `orderedResults` as alternative property names.

**Comment by tcldr** [comment]
Yeah, I can see that. How about `sequenceOutput`? A publisher already has it's associated `Output` type, so hopefully that ties things together with more clarity.

**Comment by heckj** [comment]
yeah, `sequenceOutput` seems very reasonable. In working with it a bit more today, I realized that I also would love to be able to test the data collected by the testable subscriber separate from the control messages. It's not hard to filter manually (once you clue in the subscribe and completions are listed as well), but it might be worth considering as well

**Comment by tcldr** [comment]
In the end I went with `recordedOutput` as that way it pairs nicely with `recordedDemandLog`. Thanks for the suggestions! Released in v0.6.0.
